tests/v0_tests/test_custom_vector_search.py

```diff
@@ -87,4 +87,17 @@ def test_custom_vector_search_query_format(self):
         except MarqoWebError:
             pass
 
-
+    def test_custom_context_with_wrong_dimension(self):
+        if self.IS_MULTI_INSTANCE:
+            try:
+                self.warm_request(self.client.index(self.index_name_1).search, q=self.query,
+                                  context = {"tensor": [{"vector": [1, ] * 200, "weight": 0}, {"vector": [2, ] * 512, "weight": 0}], })
+                raise AssertionError
+            except MarqoWebError as e:
```

Comment:
For warming requests, we are not sure about the state Marqo-os is in (as it is eventually consistent), so we shouldn't have assertions like this. Wait until the main body of the test to make these assertions

Reply:
thanks for pointing out, updated!